src/search/index.ts

```diff
@@ -5,8 +5,16 @@ import { Grid, Move } from '../definitions';
 import { alphaBeta } from './alphaBeta';
 import { minimax } from './minimax';
 
+let cache : Map<Grid,Number>;
```

Comment:
We need to initialize the cache, or we will get `cannot call ".get()" of "undefined"` error when we get to line #13.

Reply:
Also, the cache can be a `const` instead of a `let`, because we will never change it.